src/smoosh_utils.erl

```diff
@@ -60,20 +60,15 @@ ignore_db(DbName) when is_list(DbName) ->
 ignore_db(Db) ->
     ignore_db(couch_db:name(Db)).
 
-in_allowed_window(Channel) ->
+in_allowed_window(Time, Channel) ->
     From = parse_time(get(Channel, "from"), {00, 00}),
     To = parse_time(get(Channel, "to"), {24, 00}),
-    in_allowed_window(From, To).
-
-in_allowed_window(From, To) ->
-    {HH, MM, _} = erlang:time(),
```

Comment:
Alternative solution 1: Just update this line to `{_, {HH, MM, _}} = calendar:universal_time(),`. We could mock calendar:universal_time to test both conditions.

Reply:
Yes, as much as I dislike mocking, it's a less intrusive solution here.